app.rb

```diff
@@ -63,5 +67,17 @@ class GrootMerchService < Sinatra::Base
     disable :unsecure
   end
 
+  configure do
+    scheduler = Rufus::Scheduler.new
+    logger = Logger.new STDOUT
```

Comment:
If we're defining the logger here, then I think [this line](https://github.com/acm-uiuc/groot-merch-service/pull/19/files#diff-cc95738088603531796e0d0f246a5d77R53) can be removed, as it would overwrite it?

Can you test and make sure that's the case?